Scripts/UI/MainMenuActivator.cs

```diff
@@ -68,47 +80,72 @@ public bool activatorButtonMoveAway
 		Coroutine m_ActivatorMoveCoroutine;
 		Vector3 m_AlternateActivatorLocalPosition;
 
+		bool m_Disabled;
+		Material m_IconMaterial;
+		Color m_EnabledColor;
```

Comment:
We get enabledColor from the color the in the material. I could switch this to specifying both in the script, but generally we've been trying to avoid having a value serialized in two places, even in situations like this.